_data/projects/pablo.yml

```
@@ -0,0 +1,10 @@
+name: Pablo
+desc: A Color Palette management app
+site: https://ajsaraujo.github.io/pablo/index.html
+tags:
+  - angular
+  - website
+  - typescript
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ajsaraujo/pablo/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

```suggestion
  link: https://github.com/ajsaraujo/pablo/labels/good%20first%20issue
```
This would be better.